flight/PiOS/STM32F30x/inc/pios_ws2811_target.h

```diff
@@ -0,0 +1,53 @@
+/**
+ ******************************************************************************
+ *
+ * @file       pios_ws2811_cfg.h
```

Comment:
This needs to be `STM32F30x/inc/pios_ws2811_target.h` so it doesn't conflict with the F4 one. And _cfg.h matches our conventions (and the file contents) better?